From e485ef62632a350d146f7ea437f4ece50f9830b2 Mon Sep 17 00:00:00 2001
From: Pavel Shirshov <pavelsh@microsoft.com>
Date: Mon, 16 Nov 2020 18:11:47 -0800
Subject: [PATCH 01/10] Reduce severity of 'Vty connected from' message

---
 lib/vty.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/vty.c b/lib/vty.c
index 92db07677..daf1aca3f 100644
--- a/lib/vty.c
+++ b/lib/vty.c
@@ -1812,7 +1812,7 @@ static void vty_accept(struct thread *thread)
 		zlog_info("can't set sockopt to vty_sock : %s",
 			  safe_strerror(errno));
 
-	zlog_info("Vty connection from %pSU", &su);
+	zlog_debug("Vty connection from %pSU", &su);
 
 	vty_create(vty_sock, &su);
 }
-- 
2.25.1

